pkgs/by-name/pr/prism-model-checker-unwrapped/package.nix

```diff
@@ -0,0 +1,92 @@
+{
+  lib,
+  stdenv,
+  gccStdenv,
+  coreutils,
+  fetchFromGitHub,
+  openjdk,
+  cctools,
+  copyDesktopItems,
+  makeDesktopItem,
+}:
+let
+  stdenv' = stdenv;
+in
+let
+  stdenv = if stdenv'.hostPlatform.isDarwin then gccStdenv else stdenv';
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "prism-model-checker-unwrapped";
+  version = "4.8.1";
+
+  src = fetchFromGitHub {
+    owner = "prismmodelchecker";
+    repo = "prism";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-igFRIjPfx0BFpQjaW/vgMEnH2HLC06aL3IMHh+ELB6U=";
+  };
+
+  nativeBuildInputs = [
+    openjdk
+    copyDesktopItems
+  ] ++ lib.optionals stdenv.hostPlatform.isDarwin [ cctools ];
+
+  desktopItems = [
+    (makeDesktopItem {
+      name = "prism-model-checker-xprism";
+      desktopName = "XPrism";
+      type = "Application";
+      icon = "prism-model-checker";
+      terminal = false;
+      categories = [
+        "Science"
+        "Math"
+      ];
+      exec = "xprism";
+      comment = finalAttrs.meta.description;
```

Comment:
Meta shouldn't be referred in the deriv as it would cause a rebuild when it's changed (which is not supposed to happen)